public/js/ui.js

```diff
@@ -72,7 +84,10 @@ function initCanvas() {
 }
 
 function updateInferenceText(inference) {
-  inferenceText.innerHTML = inference.toFixed(4);
+  inferenceText.innerHTML = `${(inference * 100).toFixed(2)} %`;
```

Comment:
Do you think we really need the decimal places? Not sure if it's _that_ important :)

Reply:
I personally *prefer* having _at least_ that level of precision but I'm aware not everyone likes dealing with decimal places.
Also, the previous form was only decimal places and up to 4 so I though reducing the number from 0.1234 to 12.34% was better.

Reply:
at least for now I'm in favor of leaving it in - if nothing else, it'll give us a good gauge of how often it's re-calculating an inference (due to low chance of probability calculated for frame N and N+1 being the same up to 4 digits, relative to 2).